src/epr/main.py

```diff
@@ -45,6 +45,20 @@ def __init__(self):
         # use dispatch pattern to invoke method with same name
         getattr(self, args.command)()
 
+    def _handle_fields(self, value):
+        fields = None
+        if value is None:
+            return fields
+        if "," in value:
+            fields = [x.strip() for x in value.split(",") if x]
+        elif " " in value:
+            fields = [x.strip() for x in value.split(" ") if x]
+        elif len(value.strip()) == 26:
+            fields = [value.strip()]
```

Comment:
What's the significance of the number 26? Contextually I'd guess it's expected ID length, in which case making a const or leaving a comment may help